python/ambassador_diag/diagd.py

```diff
@@ -56,6 +60,7 @@
 __version__ = Version
 
 boot_time = datetime.datetime.now()
+tvars_cache = ExpiringDict(max_len=10, max_age_seconds=60)
```

Comment:
We need to keep previous representations of `tvars` in memory to diff between each client request.
We could theoretically support 10 clients, refreshing at least every 60 seconds -- or they'll be evicted.
This cache will be memory intensive!